bot.user.js

```diff
@@ -289,16 +289,18 @@ document.onkeydown = function(e) {
         }
 
         // Letter 'A' to increase collision detection radius
-        if (e.keyCode === 65) {
+        if (e.keyCode === 65 && window.snake.sp < 10) {
```

Comment:
what is the variable `window.snake.sp`? speed? can we pick a clearer name, or does the name come from slither.io itself?